packages/ui/src/PasswordField.tsx

```diff
@@ -87,6 +87,7 @@ export const PasswordField: FC<PasswordFieldProps> = ({
       type={visible ? 'text' : 'password'}
       inputContainerChild={overlay}
       inputClassName={cn(styles.inputContainer, inputClassName)}
+      inputIcon={Lock}
```

Comment:
We do not have it in MC. At least, currently. Should we pass the `inputIcon` as prop? Then we could keep all the stories as is and add new ones with the icon present.

Reply:
@aigoncharov Would it even make sense to use any other icon than "Lock" in here? @pawelklasa maybe we could unify this between the Cloud (lock icon on password field) and MC (no icon on password field)?

Reply:
@jozefhruska we cannot really add the Lock icon for some fields in MC as it would require adding icons for all icons of the form for consistency and it just does not make sense for complex security provider configs.
Agreed though, that it is not likely to be anything else bu the `Lock`. Should we use a boolean prop then? Say, `withIcon`?

Reply:
@aigoncharov Yep, `withIcon` prop sounds reasonable. I'll make the change 👌